### DISCUSSION TITLE
[Feature] Document section just like google photos

### DISCUSSION THREAD

**Discussion by sub4568** [immich-app/immich#14449] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Document section just like google photos as shown in image here
<img width="1153" alt="Screenshot 2024-12-02 at 6 17 00 pm" src="https://github.com/user-attachments/assets/4ece3498-75a5-46ac-a261-40ac98c0e5c6">

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can accomplish basically the same through the smart search

**Reply by sub4568** [reply]
true, however a dedicated place to access digitized receipts,screenshots etc would be great. Just a thought. Thanks

**Comment by LuckyCoders** [comment]
### Screenshots collection (Google Photos-style) + hide from main timeline

Adding a concrete ask under this Document / Collections thread (also related to closed https://github.com/immich-app/immich/discussions/28801 and https://github.com/immich-app/immich/discussions/24069).

**Problem**

Screenshots clutter the main Photos timeline. Context/CLIP search for `screenshot` (discussion 24069) can surface some of them, but it is not reliable detection and does not support hiding them from the main feed while keeping them in a dedicated place.

**What Google Photos does**

They expose a Collections entry **Screenshots and recordings**:

![Google Photos — Collections → Screenshots and recordings](https://raw.githubusercontent.com/LuckyCoders/immich/discussion-assets/google-photos-screenshots/docs/community/google-photos-screenshots-and-recordings.png)

**Proposal for Immich**

1. **Detect** screenshots (and optionally screen recordings) via path/filename heuristics at metadata extract time (e.g. `Screenshots/`, `Screenshot_*`, `Screen Shot`, Android `Captures`) — not CLIP alone — and store a flag such as `isScreenshot`.
2. **Settings option**: hide screenshots from the main timeline.
3. **Dedicated collection / filter** (like Google Photos) so they remain browsable in one place.

Happy to refine the design if maintainers prefer this under Documents (#14449) vs a separate Screenshots collection. Path-based detection would also work well for external libraries.
